app/assets/javascripts/application.js

```diff
@@ -85,6 +85,8 @@ $(function() {
   $('a[href*=#]:not([href=#])').click(function() {
     if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
       var target = $(this.hash);
+      e.preventDefault()
```

Comment:
Do you need that line for it to work? It wasn't necessary for the change that I made.

Hound is right - e isn't defined within the function, so you're calling a function on it (`preventDefault()`) without knowing where it will have an effect, because of how JavaScript handles global variables (who knows where that e is). I'm surprised that didn't throw an error in your console.